docs/src/interfacer.md

```diff
@@ -76,8 +76,34 @@ functions must be extended for all models being run.
 
 | Coupler name      | Description | Units | Default value |
 |-------------------|-------------|-------|---------------|
-| `energy` | globally integrated energy | J | `nothing` |
-| `water` | globally integrated water | kg | `nothing` |
+| `energy` | vertically-integrated energy per surface area  | J m⁻² | `nothing` |
+| `water` | vertically-integrated water per surface area | kg m⁻² | `nothing` |
+
+- `add_coupler_fields!(coupler_field_names, ::ComponentModelSimulation)`:
+A set of default coupler exchange fields is initialized for each coupled simulation,
+but depending on the component models being run, additional coupler fields may
+be required. For example, the integrated land model requires the concentration
+of atmospheric CO2 for photosynthesis calculations, but the slab ocean does not.
+`add_coupler_fields!` is extended for each component model simulation that requires
+coupler fields in addition to the defaults, allowing us to allocate space for and
+exchange the extra fields only when necessary.
+
+The default coupler exchange fields are the following, defined in
+`default_coupler_fields()` in the Interfacer module:
+| Coupler name      | Description | Units |
+|-------------------|-------------|-------|
+| `z0m_sfc` | momentum roughness length | m |
+| `z0b_sfc` | buoyancy roughness length | m |
+| `beta` | factor to scale evaporation from the surface | - |
+| `emissivity` | surface emissivity | - |
+| `F_turb_energy` | turbulent energy flux | W m⁻²  |
+| `F_turb_moisture` | turbulent moisture flux | kg m⁻²s⁻¹ |
+| `F_turb_ρτxz` | turbulent momentum flux in the x direction | kg m⁻¹s⁻² |
```

Comment:
```suggestion
| `F_turb_ρτxz` | turbulent momentum flux in the zonal direction | kg m⁻¹s⁻² |
```
(not necessarily better than x, but more commonly used for global runs, up to you)

Reply:
thank you!